packages/parcel-plugin-mdx/package.json

```diff
@@ -31,7 +31,7 @@
     "parcel-bundler": "^1.4.1"
   },
   "peerDependencies": {
-    "@mdx-js/tag": "^0.13.1-1",
+    "@mdx-js/tag": "^1.0.0-alpha.6",
```

Comment:
This should be `@mdx-js/react` (with the matching `rc` prerelease version); https://github.com/mdx-js/mdx/pull/487 moved the package.

Reply:
Thanks, I missed that.